fixtures/pytest_store.py

```diff
@@ -89,51 +89,51 @@ def base_url(self):
             else, the base_url from the config is returned."""
         return self.current_appliance.url
 
+    def _maybe_get_plugin(self, name):
+        """ returns the plugin if the pluginmanager is availiable and the plugin exists"""
+        return self.pluginmanager and self.pluginmanager.getplugin(name)
```

Comment:
instead of running into errirs i just use boolean pass-on rules

those pass on the actual objects, so None is used

since pluggy form pytest returns None for missing and blocked plugins its fine
basically the code no longer runs into `AttributeErrors`, so the handling is not needed